src/spectrecoff/Status.fs

```diff
@@ -4,20 +4,46 @@ module SpectreCoff.Status
 open System.Threading.Tasks
 open Spectre.Console
 
+type CustomSpinner =
+    { Message: string
+      Spinner: Spinner Option
+      Look: Look Option }
+
 let start statusText (operation: StatusContext -> Task<unit>) =
     task { return! AnsiConsole.Status().StartAsync(statusText, operation) }
 
-let startWithSpinner statusText spinner (operation: StatusContext -> Task<unit>) =
+let startWithCustomSpinner customerSpinner (operation: StatusContext -> Task<unit>) =
```

Comment:
Just `spinner `is fine, imo

Reply:
Renamed (hungarian notation nono, as you yourself taught me once).